docs/public/content/vuln_db/manuals/kb_importer.md

```diff
@@ -0,0 +1,181 @@
+      "affected": false
+    }
+  ],
+  "notes": [
+    {
+      "links": [
+        "https://www.bouncycastle.org/releasenotes.html",
+        "https://ubuntu.com/security/notices/USN-3727-1"
+      ],
+      "text": "In Bouncy Castle JCE Provider version 1.55 and earlier the DSA does not fully validate ASN.1 encoding of signature on verification. It is possible to inject extra elements in the sequence making up the signature and still have it validate, which in some cases may allow the introduction of 'invisible' data into a signed structure."
+    },
+    {
+      "links": [
+        "https://lists.debian.org/debian-lts-announce/2018/07/msg00009.html"
+      ],
+      "text": "It was discovered that Bouncy Castle incorrectly handled certain crypto algorithms. A remote attacker could possibly use these issues to obtain sensitive information, including private keys."
+    }
+  ]
+}
+```
+
+Information about the fields of vulnerability `metadata.json` file:
+
+ * `vulnerability_id`: Vulnerability ID.
+ * `aliases`: Aliases used for the vulnerability id.
+ * `affected_artifacts`: Affected Libraries for the vulnerability.
+ * `notes`: This is an array of `links` and `text`. Here, the links array consists of links which provide comprehensive vulnerability information and the text is the vulnerability description. (optional, it must be provided for vulnerabilities not available from the NVD).
+
+
+Commit metadata.json example file:
+```json
+{
+  "repository" : "https://github.com/bcgit/bc-java",
+  "branch" : "DEFAULT_BRANCH",
+  "timestamp" : "1476490865",
+  "commit_id" : "843c2e60f67d71faf81d236f448ebbe56c62c647"
+}
+```
+
+Information about the fields of commit metadata.json file:
+
+ * `repository`: URL of the VCS repository hosting the library project.
+ * `branch`: Branch name.
+ * `timestamp`: Timestamp when the commit was made.
+ * `commit_id`: VCS commit id.
+
+ Example run for kb-importer:
+```sh
+java -Dvulas.shared.backend.header.X-Vulas-Client-Token=<token> -Dvulas.shared.cia.serviceUrl=<cia-url> -Dvulas.shared.backend.serviceUrl=<backend-url> -jar kb-importer-3.1.14-SNAPSHOT-jar-with-dependencies.jar -d ./CVEdata/CVE-2016-1000338 -v -u -o
+```
+
+## Usage with [Project KB](https://github.com/SAP/project-kb)
+
+Import data into Eclipse Steady backend with Project KB:
+
+### Setup Project KB
+1. Install Project-KB as given in [installing-the-kaybee-tool](https://github.com/SAP/project-kb#installing-the-kaybee-tool).
+2. Run `kaybee setup` to get the kaybeeconf.yaml file.
+3. Provide the backend URL in kaybeeconf.yaml for which you want to export the data.
+
+### Import data into Steady Backend
+1. Run `kaybee import -c kaybeeconf.yaml -n 10` to export 10 Vulnerabilities from the specified backend URL. This command generates kaybee statements in `.kaybee/imported` folder. If you do not want to import data from other backend but rather create your own statement then create a statement file as shown below:
```

Comment:
This is not correct: all verbs (commands) of KayBee assume your are "sitting on KayBee", hence `import` means import into KayBee (hence "export out of the Steady backend"). This subsection should be titled "exporting vulnerability data from a Steady backend".
Also, this information is not essential for the "basic use case" of kb-importer: this is only useful once :-)

Reply:
Hi @copernico ,
In the end, I have provided a step where it says run the script to import the Vulnerability into a steady backend. That is why the title is "Import data into Steady Backend". The ultimate goal here is to import the data into a steady backend.